db/migrate/20150911195321_add_budget_roles.rb

```diff
@@ -0,0 +1,24 @@
+class AddBudgetRoles < ActiveRecord::Migration
+  def change
+    User.with_email_role_slug!(
```

Comment:
An alternative to putting this data migration in `db/migrations` would be to put it in `lib/tasks` where it would then need to be run manually as part of a deployment. I'm also fine with that, it just means one extra one-time step on deployment.